ui_sim.py

```diff
@@ -12,6 +12,8 @@
 
```

Comment:
It appears that now the player doesn't show up as an image anymore.
![image](https://github.com/Leading-AI-Development-Solutions/MAIA/assets/123680493/dfb1d6b5-8907-407e-a8a8-11247acb64cc)